docs/setup_guide.md

```diff
@@ -118,3 +131,7 @@ event ProposalQuestionCreated(
 ```
 
 There are different services available for this such as the [OpenZepplin Defender Sentinel](https://docs.openzeppelin.com/defender/sentinel).
+
+### Deploy a master copy 
+
+If the contract get an update, you can deploy a new version of a Master Copy using the hardhat task `deployMasterCopy`. An example of the command would be: `yarn hardhat --network rinkeby deployMasterCopy`
```

Comment:
Do we expect every project to deploy their own mastercopy? This would minimize the advantage of the proxy factory

Reply:
Negative, this is just for documentation purposes, if there's an update in the DAOModule contract then we should run this script

Reply:
In this case I would probably use the "hardhat-deploy" lib and use a singleton deployment. 

this will make this a little bit easier. But I don't think this is a blocker for this PR, might be something to fix in the documentation issue (#37 )

Reply:
Tackled here: https://github.com/gnosis/dao-module/pull/43